docs/installation_configuration_guide/DesignerConfiguration.md

```diff
@@ -111,13 +114,15 @@ See [development configuration](https://github.com/TouK/nussknacker/blob/staging
 
 ## Security
 
-Nussknacker has pluggable security architecture - by default we support two type of authentication: BasicAuth and
-OAuth2. You can either use default authentication provider, based on Basic authentication and static user configuration
```

Comment:
This second sentence (_you can either use default authentication provider, based on Basic Auth ..._ does not make sense in the context of having 2 (3) providers. It is probably from the old times when there was only BasicAuth supported. Alternatively, I did not understand something, hence this comment.